Tutorials/CommonUIWithLua.md

```
@@ -0,0 +1,255 @@
+# Using Common UI with Lua in LuaMachine
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)